pkg/backup/backup/backup_manager.go

```diff
@@ -407,9 +407,12 @@ func (bm *backupManager) ensureBackupPVCExist(backup *v1alpha1.Backup) (string,
 		return "ParseStorageSizeFailed", errMsg
 	}
 	backupPVCName := backup.GetBackupPVCName()
-	_, err = bm.pvcLister.PersistentVolumeClaims(ns).Get(backupPVCName)
+	pvc, err := bm.pvcLister.PersistentVolumeClaims(ns).Get(backupPVCName)
 
 	if err == nil {
+		if pvcRs := pvc.Spec.Resources.Requests[corev1.ResourceStorage]; pvcRs.Cmp(rs) == -1 {
+			return "", fmt.Errorf("%s/%s's backup pvc %s's storage size %s is less than expected storage size %s, please delete old pvc to continue", ns, name, pvc.GetName(), pvcRs.String(), rs.String())
```

Comment:
Add a short `reason` here.

Reply:
addressed in https://github.com/pingcap/tidb-operator/pull/2894/commits/f656885822777a2cc56b35542e0e13d0e539e710